docs/userGuide/components/advanced.md

```diff
@@ -36,6 +36,21 @@ You can define such a slot within the component by adding a `slot="attribute nam
 <variable name="highlightStyle">html</variable>
 </include>
 </div>
+
+<box type="tip">
+
+You may define a slot by using the shorthand notation `#attribute_name` as well! 
```

Comment:
let's encourage using this shiny new shorthand (list it upfront), then mention the deprecated syntax would still work too

Reply:
Have made the changes! :-)